components/langfuse/actions/log-trace/log-trace.mjs

```diff
@@ -0,0 +1,119 @@
+import { v4 as uuid } from "uuid";
+import app from "../../langfuse.app.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "langfuse-log-trace",
+  name: "Log Trace",
+  description: "Log a new trace in LangFuse with details. [See the documentation](https://api.reference.langfuse.com/#tag/ingestion/POST/api/public/ingestion).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The name of the trace",
+    },
+    input: {
+      type: "string",
+      label: "Input",
+      description: "The input of the trace",
+    },
+    output: {
+      type: "string",
+      label: "Output",
+      description: "The output of the trace",
+    },
+    userId: {
+      type: "string",
+      label: "User ID",
+      description: "The ID of the user",
+      optional: true,
+    },
+    sessionId: {
+      type: "string",
+      label: "Session ID",
+      description: "The ID of the session",
+      optional: true,
+    },
```

Comment:
`sessionId` should use async options.
https://api.reference.langfuse.com/#tag/sessions/GET/api/public/sessions